src/styles/gallery.scss

```diff
@@ -0,0 +1,95 @@
+.gallery {
+
+  &__images-row {
+    display: grid;
+    justify-items: stretch;
+    margin-top: $galleryRowMarginTop;
+    gap: 0 20px;
+    &--2-1 {
+      grid-template-columns: 2fr 1fr;
+    }
+    &--1-1 {
+      grid-template-columns: 1fr 1fr;
+    }
+    &--1-2 {
+      grid-template-columns: 1fr 2fr;
+    }
+    @media (max-width: $displayMiddle) {
+      grid-template-columns: 1fr 1fr;
+    }
+    @media (max-width: $displaySmall) {
+      margin-top: $galleryRowMarginTopMob;
+    }
+  }
+
+  &__photo {
+    display: block;
+    height: $galleryPhotoHeight;
+    object-fit: cover;
+    overflow: hidden;
+    border-radius: 8px;
+    @media (max-width: $displaySmall) {
+      height: $galleryPhotoHeightMob;
+    }
+  }
+
+  &__object-descr {
+    margin-top: $galleryDescrMarginTop;
+    color: $regularTextColor;
+    font-size: $reguralTextFontSize;
+    line-height: $reguralTextLineHeight;
+    @media (max-width: $displaySmall) {
+      margin-top: $galleryDescrMarginTopMob;
+    }
+  }
+
+  &__checkbox {
+    display: none;
+  }
+
+  &__hided {
+    height: 0;
+    overflow: hidden;
+    transition: height ($additionalPhotosCount * 0.5s) ease-in;
+  }
+
+  &__checkbox:checked + &__hided {
+    height: $additionalPhotosCount * (
+      $galleryRowMarginTop +
+      $galleryPhotoHeight +
+      $galleryDescrMarginTop +
+      $reguralTextLineHeight);
+    @media (max-width: $displaySmall) {
+      height: $additionalPhotosCount * (
+        $galleryRowMarginTopMob +
+        $galleryPhotoHeightMob +
+        $galleryDescrMarginTopMob +
+        $reguralTextLineHeight);
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    width: 190px;
+    padding: 20px 65px;
+    margin: 40px auto 0;
+    color: $brightColor;
+    font-size: $reguralTextFontSize;
+    font-weight: bold;
+    background-color: transparent;
+    border: 2px solid $brightColor;
+    border-radius: 8px;
+    outline: none;
+    cursor: pointer;
+    @media (max-width: $displaySmall) {
+      width: 100%;
+    }
+    &:hover {
+      opacity: 0.7;
+    }
+    & * {
+      cursor: pointer;
+    }
```

Comment:
what is this?
assign for elements you need this style in

Reply:
it is absolutely good case to use *

Reply:
it's dangerous if you in some situation decide to add an extra element in here, you have to be sure that nothing will break
this selector is no used often

Reply:
it's dangerous if you in some situation decide to add an extra element in here, you have to be sure that nothing will break
this selector is no used often

Reply:
It is button. If I will add any element in button it will cursor pointer, i think it is good decision

Reply:
if you want to assign cursor pointer to the button, then why not just assign it there, why do you use * selector just for sure?

Reply:
deleted